usr/share/rear/conf/default.conf

```diff
@@ -163,6 +163,21 @@ KERNEL_CMDLINE=""
 COPY_KERNEL_PARAMETERS=( 'net.ifnames' 'biosdevname' )
 ####
 
+####
+# The command that is called after "rear recover" succeeded to recreate the system
+# when "rear recover" is run in 'auto_recover'/'automatic' or 'unattended' mode.
+# Normally 'reboot' is called to 'reboot' the recreated system:
+RECOVERY_REBOOT_COMMAND="reboot"
```

Comment:
I really would like to see here `RECOVERY_REBOOT_COMMANDS=()` as it is compatible with the simple use case and makes complex use cases easier to implement.

Reply:
@schlomo I suppose you mean `RECOVERY_REBOOT_COMMANDS=( reboot )` at this place?

Reply:
Yes, exactly like that. Define the variable and set the default in `default.conf` like we do with everything else

Reply:
Next week I will enhance it to use
a RECOVERY_REBOOT_COMMANDS array.